cms/djangoapps/models/settings/course_details.py

```diff
@@ -25,6 +25,11 @@
     'entrance_exam_enabled',
     'entrance_exam_id',
     'entrance_exam_minimum_score_pct',
+    'about_sidebar_html',
+    'pre_enrollment_email_subject',
+    'post_enrollment_email_subject',
+    'pre_enrollment_email',
+    'post_enrollment_email',
```

Comment:
@caesar2164 This comes from changes we submitted upstream to refactor this section.